database/migrations/2021_07_13_185858_create_users_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->id();
```

Comment:
Can we just use the CID as the PK? Avoid a situation a la cts

Reply:
Fair enough - I thought about this but wasn't sure how we wanted to do it. Makes sense though

Reply:
CIDs will always be unique I guess, so would negate the need for an additional identifying column

Reply:
My thought on this - will there ever be a scenario in which we want to create a user without a CID? Or without a VATSIM registration? Potentially. Maybe. No harm keeping it like this so that we have that flexibility in the future.

I wonder if we should have a separate table for VATSIM CIDs? That way, we could have a belongs to many relationship and merge accounts in the future?

May be overthinking this but I’m not concerned at the idea of us having our own id to provide flexibility to disconnect “our id” from “their id” in the future.

Reply:
With this mechanism, that 'non-vatsim' user wouldn't be able to exist anyway as the user has a direct relationship to an authenticated VATSIM account, unless you're thinking of a different auth mechanism.

Reply:
Personally I think not being constrained by .NET IDs is nice, and it gives us some flexibility down the line if we need it. I don't see a reason why we must have CID=ID apart from convivence of just being able to assume that for the likes of queries. 
Personally I don't see much use case for one user being able to have multiple VATSIM IDs, apart from say the case where someone made a duplicate VATSIM account which then got banned I guess, but I'm not sure what benefit it would bring? I would favour sticking with as it is at the moment. If we need belongsToMany functionality later it would be a fairly straight forward DB migration (maybe a less straightforward code refactor).

Reply:
Happy to leave it as is in which case.